source/common/config/new_grpc_mux_impl.h

```diff
@@ -37,6 +37,8 @@ class NewGrpcMuxImpl : public GrpcMux,
   void pause(const std::string& type_url) override;
   void resume(const std::string& type_url) override;
   bool paused(const std::string& type_url) const override;
+  void requestAliasesResolution(const std::string& type_url,
```

Comment:
Is it reasonable to promote alias req/resp to first class in mux and subscription?

Reply:
Removed the method (pls. see the conversation above re: delta_subscription_state's requested_aliases_ field).